zingolib/src/lightclient.rs

```diff
@@ -668,6 +668,7 @@ async fn get_recent_median_price_from_gemini() -> Result<f64, PriceFetchError> {
 }
 
 #[cfg(test)]
+#[cfg(feature = "test-features")]
```

Comment:
From a look at the rust reference, I believe that multiple cfg attributes are additive. This would be much clearer if it's a single `#[cfg(all(test, feature = "test-features"))]`

Reply:
I'm starting to think that I'm wrong and `#[cfg(any(test, feature = "test-features"))]` is correct, in which case that would still also be much clearer!